strictTsConfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "downlevelIteration": true,
+    "lib": ["es2017", "dom", "webworker"],
+    "outDir": "local_compiled_js_for_test",
+    "rootDir": ".",
+    "skipLibCheck": true,
+    "target": "es5",
+    "sourceMap": true,
+    "strict": true,
+    "typeRoots": ["./node_modules/@types"],
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "baseUrl": ".",
+    "paths": {
+      "app.constants": ["core/templates/app.constants"],
+      "app-type.constants": ["core/templates/app-type.constants"],
+      "components/*": ["core/templates/components/*"],
+      "domain/*": ["core/templates/domain/*"],
+      "expressions/*": ["core/templates/expressions/*"],
+      "pages/*": ["core/templates/pages/*"],
+      "services/*": ["core/templates/services/*"],
+      "classifiers/*": ["extensions/classifiers/*"],
+      "interactions/*": ["extensions/interactions/*"],
+      "filters/*": ["core/templates/filters/*"],
+      "static/*": ["third_party/static/*"],
+      "tests/*": ["core/templates/tests/*"]
+    }
+  },
+  "files": [
+    "core/templates/pages/admin-page/services/admin-router.service.ts",
+    "core/templates/pages/admin-page/services/admin-router.service.spec.ts"
+  ],
+  "include": ["typings/*.ts"]
```

Comment:
Just `typings` should work too, right? Also, should we include `core/templates/services/contextual` since these seemed to work okay.

Reply:
Actually there is a test folder inside typings that imports other files.

Reply:
I tried adding `core/templates/services/contextual` there were some errors thrown.

Reply:
Ah, okay, just realized I only tried it with `strictNullChecks`.

Reply:
> Actually there is a test folder inside typings that imports other files.

Should we also modify tsconfig.json then?

Reply:
Umm, why. That passes normal checks, right?